js/core.js

```diff
@@ -1709,7 +1709,13 @@ var ERMrest = (function (module) {
          * @returns {string} The formatted value.
          */
         this.formatvalue = function (data, options) {
-            if (data === undefined || data === null) {
+            //This check has been added to show "null" in all the rows if the user inputs blank string
+            if(this.type.name!=='json'){
```

Comment:
You can rewrite these two `if` statements like this:
``` javascript
if (data === undefined || (data === null && this.type.name.indexOf('json') !== 0) {
  //...
}
```
It would be easier to understand and also supports `jsonb`. 

Also add a comment that why we're opting `json` out in this case.